homeworks/Dmitriy.Tyulpa_innocentDimon/homework_7/task.js

```diff
@@ -0,0 +1,15 @@
+function fibonachi() {
+    let a = 0;
+    let b = 1;
+    const random = Math.floor(Math.random() * 50 + 50);
+    const result = [a, b];
+    for (let i = 3; i <= random; i++) {
+        const c = a + b;
+        a = b; b = c;
+        result.push(b);
```

Comment:
your loop will run `random` amount iterations, but it should actually stop when your Fibo number is more than maximum number allowed